cfp_crawler/proposal/items.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+# Define here the models for your scraped items
+#
+# See documentation in:
+# https://doc.scrapy.org/en/latest/topics/items.html
+
+import scrapy
+
+
+class ProposalItem(scrapy.Item):
+    # define the fields for your item here like:
+    # name = scrapy.Field()
```

Comment:
@Niveshkrishna, You are not declaring any item in ```items.py```. Why?

Reply:
Unless there are multiple crawlers which are using the same item, there's not much of difference in using the items and normal variables. Since there is only one crawler, it is okay to have them declared directly in `crawler.py` file. Or is it not? Is there any advantage of using items in this case?

Reply:
You are right. But it would help to reuse a bunch of code in future and to keep everything tidy, clean and understandable.